common/components/Footer/index.tsx

```diff
@@ -198,12 +214,16 @@ export default class Footer extends React.PureComponent<Props, State> {
             </div>
 
             <div className="Footer-support-donate">
-              <div className="Footer-support-donate-currency">Donate ETH</div>
+              <div className="Footer-support-donate-currency">
+                {translateRaw('DONATE_CURRENCY') + ' ETH'}
```

Comment:
This is a case for variables, some languages may put the subject (ETH) before the verb (Donate)